plugins/authentication/RestfulAuthenticationManager.php

```diff
@@ -205,5 +209,20 @@ protected function getOriginalUserSession() {
     return $this->originalUserSession;
   }
 
-
+  /**
+   * Set the user's last access time.
+   *
+   * @param object $user
+   *   A drupal user account.
+   *
+   * @see _drupal_session_write()
+   */
+  protected function setAccessTime($user) {
+    // This logic is pulled directly from _drupal_session_write().
```

Comment:
While I do reference the ```_drupal_session_write()``` in the docblock, I wanted to explicitly call out what was being pulled from the function in case more logic is added to this method in the future.